src/main/webapp/index.html

```diff
@@ -11,7 +11,7 @@
  <body>
     <div id="content">
```

Comment:
I checked the raw content of this file against https://validator.w3.org/nu/ and there are some errors for the HTML file; can you go through and make sure the HTML is valid?